evap/contributor/templates/contributor_course_form.html

```diff
@@ -92,6 +94,20 @@
     </form>
 {% endblock %}
 
+{% block modals %}
+    {{ block.super }}
+    {% trans "Approve course" as title %}
+    {% trans "If you approve this course you won't be able to make any additional changes afterwards, unless the student representatives reopen the course for changes.<br /><strong>Please approve the course when you're done editing it.</strong>" as question %}
```

Comment:
why not `If you approve blah blah. Are you sure you're done editing it.?` to make clear what this modal want from you or `Are you sure you want to approve it?` to be in line with the approval button's text?

Or something like `Are you sure you want to approve this course? This will allow the student representatives to proceed with its evaluation, but you won't be able to make any further changes.`

about the reopening part, advertising it basically tells them "nevermind if you make mistakes, we can reopen them anytime", not sure if that is what you want.